x-pack/plugins/stack_alerts/server/alert_types/es_query/alert_type.ts

```diff
@@ -119,6 +116,29 @@ export function getAlertType(logger: Logger): RuleType<
     }
   );
 
+  const actionVariableSearchTypeLabel = i18n.translate(
+    'xpack.stackAlerts.esQuery.actionVariableContextSearchTypeLabel',
+    {
+      defaultMessage: 'The type of search.',
```

Comment:
Do we think the user needs to know what type of search this rule is in their action notifications?

Reply:
I don't think this is usually necessary, but you never know, there might be a case. So I think we should keep it.

Reply:
I think we can omit it for the user.